docs/toc.html

```diff
@@ -27,6 +27,9 @@
                 <li><a href="#quickstart">1.3 Quick Start</a>
                 <li><a href="#ecosystem">1.4 Ecosystem</a>
                 <li><a href="#upgrade">1.5 Upgrading</a>
+                    <ul>
+                        <li><a href="/{{version}}/zk2kraft">1.5.1 Removed Zookeeper Configurations and Metrics</a>
```

Comment:
@frankvicky Just a tiny suggestion to improve clarity: 
```suggestion
                        <li><a href="/{{version}}/zk2kraft">1.5.1 Removing Zookeeper Configurations and Metrics</a>
```
Kafka documentation style:
![image](https://github.com/user-attachments/assets/fc66c042-5dbb-4346-aab0-eac065a4c034)

Reply:
I think "Removed" is correct for changelogs describing completed changes. "Removing" here describes an operation. This is not the same context.

Reply:
why not using the title "Differences Between KRaft mode and Zookeeper mode"?

Reply:
Also, could you move it to "1.6" or "1.7" instead of "1.5.1"?

Reply:
How about this?
![image](https://github.com/user-attachments/assets/4de29538-7507-4301-a7e3-0a830fd6bd87)

Reply:
![image](https://github.com/user-attachments/assets/952b6c0a-ded3-4d53-b754-7a7060acf2e3)